src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+form {
+  margin-top: 20px;
+}
+
+input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.search {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px 19px;
+  background-position: 2% 50%;
+  padding-left: 62px;
+}
+
+.search--small {
+  height: 42px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 19px;
+  background-size: 11px 11px;
+  background-position: 1% 50%;
+  padding-left: 33px;
+}
```

Comment:
you have some style duplications here. add class `search` and one additional class for small search to change only needed styles
![image](https://user-images.githubusercontent.com/22824431/235138569-eecf71b3-4963-4abc-9281-ee6ddabf9ee6.png)

Reply:
fix all cases